app/lib/filters/dropdown_select_filter.rb

```diff
@@ -0,0 +1,17 @@
+module Filters
+  class DropdownSelectFilter < Filter
```

Comment:
A very minor point but perhaps we should mirror the HTML naming convention and call this `SelectFilter`. `Dropdown` alludes to specific behaviour if the element has a size of 1.